examples/mnist/quantized.cpp

```diff
@@ -45,6 +45,7 @@ void construct_net(network<sequential> &nn) {
      << quantized_convolutional_layer<tan_h>(
           5, 5, 5, 16, 120, padding::valid, true, 1, 1,
           backend_type)  // C5, 16@5x5-in, 120@1x1-out
+     << tanh_layer(120)
```

Comment:
New `tanh_layer` layer is added here but the previous node still has `<tan_h>` template parameter.